### PR TITLE
EDU-626 Replace action links on docs page with icons

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-626

### CODE REVIEW

File: src/components/pages/docs.less [hunk 1]
```diff
@@ -17,6 +17,49 @@
...
+  .m-box-shadow;
+  background-color: @edu-white;
+  width: @edu-control-height-sm;
+  height: @edu-control-height-sm;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+  opacity: 90%;
+  cursor: pointer;
```
Comment (adrianaluchian) [inline src/components/pages/docs.less:37]: Tiny thing: this is redundant.

**Comment by adrianaluchian** [comment]
This small change makes such a big difference in the UI 😳 The page no longer looks like an "admin-only" page.

**Comment by ahelmberger** [comment]
> This small change makes such a big difference in the UI flushed The page no longer looks like an "admin-only" page.

I thought the same, the page looks a lot less bureaucratic.

File: src/components/pages/docs.less (hunk 1 above)
Comment (ahelmberger) [reply src/components/pages/docs.less]: True, removed. Thanks!

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**